tests/data/siegel_data.py

```diff
@@ -169,83 +160,38 @@ def sectional_curvature_test_data(self):
         smoke_data = [
             dict(
                 n=2,
-                scale=1.0,
                 tangent_vec_a=[[1.0, 0.0], [0.0, 0.0]],
                 tangent_vec_b=[[0.0, 0.0], [0.0, 1.0]],
                 base_point=[[0.0, 0.0], [0.0, 0.0]],
                 expected=0,
             ),
             dict(
                 n=2,
-                scale=1.0,
                 tangent_vec_a=[[1j, 0j], [0j, 1j]],
                 tangent_vec_b=[[-1j, 0j], [0j, -1j]],
                 base_point=[[0j, 0j], [0j, 0j]],
                 expected=0,
             ),
             dict(
                 n=2,
-                scale=1.0,
                 tangent_vec_a=[[1.0 + 0j, 0j], [0j, 0j]],
                 tangent_vec_b=[[0j, 1j], [0j, 0j]],
                 base_point=[[0j, 0j], [0j, 0j]],
                 expected=-1,
             ),
             dict(
                 n=2,
-                scale=1.0,
                 tangent_vec_a=[[2 + 0j, 0j], [0j, 0j]],
                 tangent_vec_b=[[0j, 2j], [0j, 0j]],
                 base_point=[[0j, 0j], [0j, 0j]],
                 expected=-1,
             ),
-            dict(
-                n=2,
-                scale=2.0,
-                tangent_vec_a=[[0.25 + 0j, 0j], [0j, 0j]],
-                tangent_vec_b=[[0j, 0.25j], [0j, 0j]],
-                base_point=[[0j, 0j], [0j, 0j]],
-                expected=-0.25,
-            ),
```

Comment:
could we keep this by removing scale and changing expected accordingly? (just because I don't see this case without scale and it is nice to keep smoke data.)

Reply:
Retained scale by using the parameter in the test file as `scale * self.Metric(n)`.

Reply:
@johnharveymath, I would prefer here to fully remove `scale` from `Siegel` tests. Other tests care about the influence of the scale (e.g. tests on any child of `ProductManifold`). The other manifold tests should "ignore" the existence of `ScalarProductMetric`.

Reply:
Is this particular to `Siegel`? There is `ComplexPoincareDisk` as well, where there are some `scale` parameters in the smoke data.

Reply:
In the latest commits I've done it for both classes.